runtime/vm/vmthread.c

```diff
@@ -558,7 +558,14 @@ threadParseArguments(J9JavaVM *vm, char *optArg)
 #endif /* defined(OMR_THR_YIELD_ALG) */
 	
 #if defined(LINUX)
-	/* if Completely Fair Scheduler is detected, and sched_compat_yield=0, default to -Xthr:minimizeUserCPU */
+	/* Settings for the three-tier spinlock loops if the Completely Fair Scheduler (CFS) is detected,
+	 * and sched_compat_yield=0. sched_compat_yield=1 enables the aggressive CPU yielding behavior of
+	 * the O(1) scheduler by moving the relinquishing task to the end of the runnable queue. For
+	 * sched_compat_yield=1, the yield count is set to 45 in the three-tier spinlock loops. For the
+	 * CFS and sched_compat_yield=0, the CPU yielding behavior is moderate in comparison to the CPU
+	 * yielding behavior of the O(1) scheduler. So, the yield count is increased to 270 in the
+	 * three-tier spinlock loops.
+	 */
```

Comment:
I had trouble parsing this and its applicability to the code below.  Maybe something like:
```c
/* Check for the sched_compat_yield setting for versions of the the Complete Fair Scheduler (CFS)
 * which have broken thread_yield behaviour.  If running in CFS and sched_compat_yield=0, the CPU
 * yielding behavior is moderated to act as though CFS was not enabled.  Increase the yield count
 * to 270 in the three-tier spinlock loops.
 * 
 * sched_compat_yield=1 uses the aggressive CPU yielding behavior of some versions of the O(1) 
 * scheduler and uses the default yield counts.
 * 
 * Newer versions of linux have removed this compat flag as the thread_yield behaviour has been
 * restored to something sane.
 */
```

Reply:
@DanHeidinga I have updated the comment.